openhands/cli/commands.py

```diff
@@ -882,3 +890,306 @@ async def remove_mcp_server(config: OpenHandsConfig) -> None:
             restart_cli()
     else:
         print_formatted_text(f'Failed to remove {server_type} server "{identifier}".')
+
+
+def show_loading_spinner(stop_event: threading.Event) -> None:
+    """Show a loading spinner animation."""
+    spinner_chars = ['⠋', '⠙', '⠹', '⠸', '⠼', '⠴', '⠦', '⠧', '⠇', '⠏']
```

Comment:
@OpenHands i think we have spinner implementation somewhere else, can you deduplicate?